package.json

```diff
@@ -28,7 +34,9 @@
     "lint": "eslint --ext .js --ext .vue src",
     "docs": "vue-docgen -c docgen.config.js",
     "build": "vue-cli-service build --target lib --formats umd-min --name VueXIVTooltips --filename vue-xivtooltips src/index.js",
-    "build:dev": "vue-cli-service build --mode development --target lib --formats umd --name VueXIVTooltips --filename vue-xivtooltips src/index.js"
+    "build:dev": "vue-cli-service build --mode development --target lib --formats umd --name VueXIVTooltips --filename vue-xivtooltips src/index.js",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
Are these 2 actual tasks defined somewhere, like does storybook CLI add the commands?

Reply:
the storybook setup adds those commands, but they do work. I forgot to add a babel package removed during the cleanup, but ultimately not terrible.

Reply:
Just for clarity, these get installed to `node_modules/.bin` which yarn scripts have in their exec path.